### PR TITLE
build: install python-dbusmock via pip for a Docker container

### PR DESCRIPTION
**Pull request by alexeykuzmin**
#### Description of Change
To prevent timeout errors with some other python-dbusmock versions.

Related to #18523.
* Linux builds are timing out but passing tests. It turns out that dbusmock was hanging on dbus_stop. This PR changes the docker container to use a newer version of python-dbusmock which adds timeouts to dbus_stop.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: no-notes

### CODE REVIEW

**Comment by codebytere** [comment]
@alexeykuzmin what timeout errors were we seeing? do you have links to relevant tests that were dying, by chance?

**Comment by alexeykuzmin** [comment]
@codebytere see the second bullet point in the #18523 linked in the description.

**Comment by alexeykuzmin** [comment]
@jkleinsc I remember you saying that all containers for branches > 3 work fine,
but if we'll have to build a new container version, it will get `python-dbusmock` 0.17.0-1 with which I see that timeout issue.

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,9 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 # crcmod is required by gsutil, which is used for filling the gclient git cache
 RUN pip install -U crcmod
 
+# dbusmock is needed for Electron tests
+RUN pip install python-dbusmock==0.18.1
```
Comment (jkleinsc) [inline Dockerfile]:
```suggestion
RUN pip install python-dbusmock
```

**Pull request merged**